map/discovery/discovery_manager.cpp

```diff
@@ -31,7 +31,6 @@ search::SearchParams Manager::GetSearchParams(Manager::Params const & params, It
 {
   search::SearchParams p;
   p.m_query = GetQuery(type);
-  p.m_inputLocale = params.m_lang;
   p.m_viewport = params.m_viewport;
```

Comment:
Why not to explicitly specify that input_locale is "en" ?

Reply:
Do we need to do this? As far as I got you it's "en" by default.

Reply:
Ok now I see.